sortinghat/core/errors.py

```diff
@@ -87,10 +87,10 @@ class ClosedTransactionError(BaseError):
 
 
 class LockedIdentityError(BaseError):
```

Comment:
My idea was to keep it with that name at least for now. The reason is we want to have multiple ids to access the individuals. Those ids will be the uuids of the identities so that's what I thought to keep that name until we have more clear what to do. What do you think? Can we leave this name for now and change it later?

Reply:
Ok, I think we can keep it like that for now. We can say an identity is locked as it belongs to an individual which is also locked, that's why we get a `LockedIdentityError`, so I think it makes sense. But I do think that this should be renamed at some point if what is locked is the "Individual" object, even if all the related objects become "locked" too.